src/mediaEditor/plugins/mediaAlignment/helper.js

```diff
@@ -43,12 +43,17 @@ export const positionFloat = function positionFloat(widget, position) {
     // Update DOM
     widget.$container.addClass(className);
     // Update model
+    const prevClassName = widget.element.attr('class');
     if (className) {
         widget.element.attr('class', className);
     } else {
         widget.element.removeAttr('class');
     }
-
+    
+    if ((prevClassName || className.length) && prevClassName !== className) {
```

Comment:
**question:** Why test the value only for one variable and the length only for the other variable? Both are string, so it could be tested by value for both...

**suggestion:**
```suggestion
    if ((prevClassName || className) && prevClassName !== className) {
```